diff --git a/Sources/OpenAPI/Generated/Entities/AppStoreVersionUpdateRequest.swift b/Sources/OpenAPI/Generated/Entities/AppStoreVersionUpdateRequest.swift
index ebb61c7..b41caa9 100644
--- a/Sources/OpenAPI/Generated/Entities/AppStoreVersionUpdateRequest.swift
+++ b/Sources/OpenAPI/Generated/Entities/AppStoreVersionUpdateRequest.swift
@@ -158,7 +158,7 @@ public struct AppStoreVersionUpdateRequest: Codable {
 
 				public func encode(to encoder: Encoder) throws {
 					var values = encoder.container(keyedBy: StringCodingKey.self)
-					try values.encodeIfPresent(data, forKey: "data")
+					try values.encode(data, forKey: "data")
 				}
 			}
 
