diff --git a/Sources/AppStoreConnectApi/OpenAPI/Generated/Entities/CiBuildRun.swift b/Sources/AppStoreConnectApi/OpenAPI/Generated/Entities/CiBuildRun.swift
index aa6cf8089bd6b3548c8e7862d920aa07c64060c4..51711a2e789ff99fab7eec9bb1f35744131d1dbc 100644
--- a/Sources/AppStoreConnectApi/OpenAPI/Generated/Entities/CiBuildRun.swift
+++ b/Sources/AppStoreConnectApi/OpenAPI/Generated/Entities/CiBuildRun.swift
@@ -33,9 +33,9 @@ public struct CiBuildRun: Codable, Identifiable {
 			public var message: String?
 			public var author: CiGitUser?
 			public var committer: CiGitUser?
-			public var webURL: URL?
+			public var webURL: String?
 
-			public init(commitSha: String? = nil, message: String? = nil, author: CiGitUser? = nil, committer: CiGitUser? = nil, webURL: URL? = nil) {
+			public init(commitSha: String? = nil, message: String? = nil, author: CiGitUser? = nil, committer: CiGitUser? = nil, webURL: String? = nil) {
 				self.commitSha = commitSha
 				self.message = message
 				self.author = author
@@ -49,7 +49,7 @@ public struct CiBuildRun: Codable, Identifiable {
 				self.message = try values.decodeIfPresent(String.self, forKey: "message")
 				self.author = try values.decodeIfPresent(CiGitUser.self, forKey: "author")
 				self.committer = try values.decodeIfPresent(CiGitUser.self, forKey: "committer")
-				self.webURL = try values.decodeIfPresent(URL.self, forKey: "webUrl")
+				self.webURL = try values.decodeIfPresent(String.self, forKey: "webUrl")
 			}
 
 			public func encode(to encoder: Encoder) throws {
@@ -67,9 +67,9 @@ public struct CiBuildRun: Codable, Identifiable {
 			public var message: String?
 			public var author: CiGitUser?
 			public var committer: CiGitUser?
-			public var webURL: URL?
+			public var webURL: String?
 
-			public init(commitSha: String? = nil, message: String? = nil, author: CiGitUser? = nil, committer: CiGitUser? = nil, webURL: URL? = nil) {
+			public init(commitSha: String? = nil, message: String? = nil, author: CiGitUser? = nil, committer: CiGitUser? = nil, webURL: String? = nil) {
 				self.commitSha = commitSha
 				self.message = message
 				self.author = author
@@ -83,7 +83,7 @@ public struct CiBuildRun: Codable, Identifiable {
 				self.message = try values.decodeIfPresent(String.self, forKey: "message")
 				self.author = try values.decodeIfPresent(CiGitUser.self, forKey: "author")
 				self.committer = try values.decodeIfPresent(CiGitUser.self, forKey: "committer")
-				self.webURL = try values.decodeIfPresent(URL.self, forKey: "webUrl")
+				self.webURL = try values.decodeIfPresent(String.self, forKey: "webUrl")
 			}
 
 			public func encode(to encoder: Encoder) throws {
