diff --git a/Sources/AppStoreConnectApi/OpenAPI/Generated/Entities/CiGitUser.swift b/Sources/AppStoreConnectApi/OpenAPI/Generated/Entities/CiGitUser.swift
index d76a2e9c7e65890f240d41b12f11dcafaf3ca287..1444fbb128a23289e202bbd45d941ed43be87fa3 100644
--- a/Sources/AppStoreConnectApi/OpenAPI/Generated/Entities/CiGitUser.swift
+++ b/Sources/AppStoreConnectApi/OpenAPI/Generated/Entities/CiGitUser.swift
@@ -5,9 +5,9 @@ import Foundation
 
 public struct CiGitUser: Codable {
 	public var displayName: String?
-	public var avatarURL: URL?
+	public var avatarURL: String?
 
-	public init(displayName: String? = nil, avatarURL: URL? = nil) {
+	public init(displayName: String? = nil, avatarURL: String? = nil) {
 		self.displayName = displayName
 		self.avatarURL = avatarURL
 	}
@@ -15,7 +15,7 @@ public struct CiGitUser: Codable {
 	public init(from decoder: Decoder) throws {
 		let values = try decoder.container(keyedBy: StringCodingKey.self)
 		self.displayName = try values.decodeIfPresent(String.self, forKey: "displayName")
-		self.avatarURL = try values.decodeIfPresent(URL.self, forKey: "avatarUrl")
+		self.avatarURL = try values.decodeIfPresent(String.self, forKey: "avatarUrl")
 	}
 
 	public func encode(to encoder: Encoder) throws {
